apps/server/src/modules/h5p-editor/contentStorage/contentStorage.spec.ts

```diff
@@ -263,36 +376,47 @@ describe('ContentStorage', () => {
 		describe('WHEN content is deleted successfully', () => {
```

Comment:
This describe doesn't make sense because it forecasts the result of the test. (I know there are a lot of places in our code where it is used like this... :/ ) 

It's just the normal flow of the function and no special scenario. So IMO this test shouldn't be enclosed in a surplus describe.